container/webui/docker-compose.yaml

```diff
@@ -90,6 +90,7 @@ services:
       MOJO_LISTEN: "http://0.0.0.0:9526"
       OPENQA_SCHEDULER_HOST: "scheduler"
       OPENQA_WEB_SOCKETS_HOST: "websockets"
+      MOJO_CLIENT_DEBUG: 1
```

Comment:
This isn't supposed to stay when we merge the PR, right?

Reply:
No, you have to consider that this is still not-ready